tests/sqla_models_tests.py

```diff
@@ -54,10 +54,11 @@ def test_is_time_druid_time_col(self):
 
         database = Database(database_name="druid_db", sqlalchemy_uri="druid://db")
         tbl = SqlaTable(table_name="druid_tbl", database=database)
-        col = TableColumn(column_name="__time", type="INTEGER", table=tbl)
-        self.assertEqual(col.is_dttm, None)
-        DruidEngineSpec.alter_new_orm_column(col)
-        self.assertEqual(col.is_dttm, True)
+        column_spec = DruidEngineSpec.get_column_spec(column_name="__time")
+        print(">>> {}".format(column_spec))
+        self.assertTrue(column_spec.is_dttm)
```

Comment:
The purpose of this test is to make sure that a column with the name `__time` and `type="INTEGER"` (=the non-temporal type that comes back from the db) is interpreted as a temporal column and gets the correct sqla type. Therefore it's important to make sure that the type is defined in the `TableColumn` instantiation.

Also, some debugging leftover:
```suggestion
        column_spec = DruidEngineSpec.get_column_spec(column_name="__time")
        self.assertTrue(column_spec.is_dttm)
```